temporalio/converter.py

```diff
@@ -133,7 +154,7 @@ def __init__(self, *converters: PayloadConverter) -> None:
         # Insertion order preserved here
         self.converters = {c.encoding.encode(): c for c in converters}
```

Comment:
```suggestion
        self.converters = {c.encoding.encoding: c for c in converters}
```

Reply:
`encoding` is a `str`, and `encode()` is https://docs.python.org/3/library/stdtypes.html#str.encode to convert to `bytes` which matches the metadata value type which is a better key for this dict.

Reply:
Sorry, that's what happens when I review this late